.github/workflows/publish.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".changeset/**"
+      - ".github/workflows/publish.yml"
```

Comment:
It doesn't seem like this is needed. I checked out a few repos, none of them have such restrictions

Reply:
Yeah, this is not exactly needed. I removed it